src/package.xml

```diff
@@ -1,104 +1,44 @@
 <?xml version="1.0" encoding="utf-8"?>
-<package name="helloworldTimePy" displayName="Example demonstrating how to use timesteps with a python model" version="1.0.0" url="https://apexrms.github.io/helloworldEnhanced/">
-	<transformers>
+<package name="helloworldTimePy" displayName="Hello World Time (Python)" description = "Example demonstrating how to use timesteps with a python model" version="2.0.0" url="https://apexrms.github.io/helloworldEnhanced/">
 
-		<!--Primary Transformer-->
-		<transformer
-			name="Primary"
-			isPrimary="True"
-			programName="python"
-			programArguments="model.py">
+	<!--Scenario Datasheets-->
 
-			<!--Chart Transformer-->
-			<include>
-				<transformer name="corestime_Runtime"/>
-			</include>
+	<!--Run Control-->
+	<dataSheet name="RunControl" displayName="Run Control" isRunControl="True" timestepTitle="Timestep" showMinIteration="False"/>
 
-			<datafeeds>
+	<!--Model Inputs-->
+	<dataSheet name="InputDatasheet" isSingleRow="True">
```

Comment:
Can you add display names for the input/output datasheets? Maybe "Inputs" and "Outputs".

Reply:
Can you also change the displayName for "Outputs"?